package.json

```diff
@@ -11,18 +11,13 @@
     "node": "14.x"
   },
   "scripts": {
-    "compile": "yarn run truffle compile",
-    "start": "yarn run compile && yarn run start:client",
-    "start:client": "yarn workspace client run start",
-    "lint": "yarn run lint:prettier && yarn run lint:solhint",
-    "lint:prettier": "yarn prettier --check .",
-    "lint:solhint": "yarn solhint ./**/*.sol",
-    "fix": "yarn run fix:prettier && yarn run fix:solhint",
-    "fix:prettier": "yarn prettier --write .",
-    "fix:solhint": "yarn solhint --fix ./**/*.sol",
-    "build": "yarn run compile && yarn run build:client",
-    "build:client": "yarn workspace client run build",
-    "test": "yarn workspace client run test"
+    "start": "cd client && npm run start",
+    "postinstall": "cd client && npm install",
+    "lint": "prettier --check .",
+    "fix": "prettier --write .",
+    "prebuild": "./node_modules/truffle/build/cli.bundled.js compile",
```

Comment:
We can take this out for now since we're not doing any truffle related stuff, right?